### DISCUSSION TITLE
Add more layouts and structured data

### DISCUSSION THREAD

**Discussion by blogi-kriptovalyut** [mmistakes/minimal-mistakes#2687] [Ideas]
Hello.

## Summary
Add different types of structured data, not just Person, Organization and CreativeWork
## Motivation
Implementing structured data on your site is also a way to prepare for the future of search, as Google in particular continues to move in the direction of hyper-personalization and solving problems and answering questions directly.
I see the following benefits:
• Higher CTR
• More conversions
• Authority
Google, Bing, and other search engines encourage webmasters to use structured data, and incentivize that usage by providing benefits to websites with structured data correctly implemented.
This can be done in the `_layout`s folder.
The logic is this:
• Material type: [Article](https://developers.google.com/search/docs/data-types/article)
• `_layout`: articles and in this file make the markup for the "`Article`" dictionary

Material Type: [how-to](https://developers.google.com/search/docs/data-types/how-to)
• `_layout`: how-to and in this file make markup for the how-to dictionary

It would be great to mark up the [faqpage](https://developers.google.com/search/docs/data-types/faqpage), [factcheck](https://developers.google.com/search/docs/data-types/factcheck) critic review, etc.

I think structured date becomes as important as Mobile first at one time.

What do you think?

**Comment by mmistakes** [comment]
My thought was to replace all SEO related concerns with the official **`jekyll-seo-tag` plugin** (see #324), which [`_includes/seo.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/seo.html) is heavily inspired by.

The official Jekyll plugin has more features and allows you to [customize the JSON-LD output](https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/advanced-usage.md#customizing-json-ld-output) to some degree. It makes sense to leverage the plugin and the community. But as this is a free theme and my time is extremely limited I haven't gotten around to it.

Perhaps some day...
